tests/test_create.py

```diff
@@ -379,6 +379,7 @@ def test_node_and_cluster_settings_may_override(self):
                     "auth.host_based.enabled": "node-override",
                     "node.attr.node_setting": "node-override",
                     "node.attr.some_node_setting": "node",
+                    "node.attr.zone": "$(curl -s 'https://example.com')",
```

Comment:
I don't think the change in this test is valuable. There's no override happening, since the operator is not run with any `CLOUD_PROVIDER` setting during tests.

Reply:
We have 2 code paths where `node.attr.zone` gets special treatment, so it felt like a good idea to also test the case where it doesn't get any special treatment. But ok, removed it.